src/SamplesApp/SamplesApp.UITests/Windows_UI_Core/CoreDispatcher/CoreDispatcher_Tests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Core.CoreDispatcher
+{
+  [TestFixture]
+  public partial class CoreDispatcher_Tests : SampleControlUITestBase
+  {
+	[Test]
+	[AutoRetry]
+	public void RunAsyncExtensions()
+	{
+	  Run("UITests.Shared.Windows_UI_Core.CoreDispatcherControl.CoreDispatcherControl");
+
+	  _app.Marked("RunAsyncButton").Tap();
+
+
+	  Thread.Sleep(1200);
+
+	  _app.Marked("Result").WithText("Success");
```

Comment:
You can replace this pattern by
```csharp
_app.WaitForDependencyPropertyValue(_app.Marked("Result"), "Text", "Success");
```

So you won't need to introduce an abritary waiting delay...

Reply:
As per Jerome's request, I've moved my test to the Runtime tests section and removed this UI test. But thanks for the tip!